src/journal/models.py

```diff
@@ -459,6 +464,15 @@ class Issue(models.Model):
         through='IssueEditor',
     )
 
+    code = models.CharField(
```

Comment:
Could this not just be a slugfield for simplicity?

Reply:
> Could this not just be a slugfield for simplicity?

yep, that works. I always think of SlugField as an auto-generated field, but it doesn't need to be